libshaderc/include/shaderc/shaderc.h

```diff
@@ -86,6 +86,93 @@ typedef enum {
   shaderc_optimization_level_size,  // optimize towards reducing code size
 } shaderc_optimization_level;
 
+// Resource limits.
+typedef enum {
```

Comment:
Yeah.  This is like circularity we've seen before:
* the public header is in libshaderc/shaderc.h
* but libshaderc_util code has to talk about those enums to translate down into Glslang.
So we need two mirrored enum sets.  The best I can do is the single resources.inc file that makes the association.